app/api/items/leave.go

```diff
@@ -0,0 +1,113 @@
+package items
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/go-chi/render"
+
+	"github.com/France-ioi/AlgoreaBackend/app/database"
+	"github.com/France-ioi/AlgoreaBackend/app/service"
+)
+
+// swagger:operation DELETE /attempts/{attempt_id} items itemAttemptEnd
+// ---
+// summary: Leave the item
```

Comment:
`itemAttemptEnd`<> "Leave the item" , `ended_at` fct <> `leave.go`
We should decide if it is leave or end. I know I proposed "leave" in the issue, but intuitively when reading this service and as the attribute is `ended_at`, I find `end` clearer.

Reply:
fixed